ros2/README.md

```diff
@@ -0,0 +1,185 @@
+# ROS 2 Bindings
+
+This document defines how to describe ROS 2-specific information in AsyncAPI.
+
+It applies to all versions of ROS 2 (foxy, galactic, humble, iron, jazzy).
+
+<a name="version"></a>
+
+## Version
+
+Current version is `0.1.0`.
+
+<a name="server"></a>
+
+## Server Binding Object
+
+This object contains information about the server representation in ROS 2. 
+ROS 2 can use either DDS or Zenoh as its middleware. 
```

Comment:
Minor perhaps, but while the main RMWs in use may be DDS-based, there are certainly others that aren't (`rmw_zenoh` being a recent addition, but also [eclipse-ecal/rmw_ecal](https://github.com/eclipse-ecal/rmw_ecal) fi).

Reply:
Thank you for your comment!

As you know, there are a quite  acouple of them with my favourite (for the sake of this exact argument) the [christophebedard/rmw_email](https://github.com/christophebedard/rmw_email).

We also had this argument earlier with @fmvilas , that for the moment we only want to focus on DDS and Zenoh based systems. But I think that the majority of the RMW implementations should also be representable through this binding. But this current implication should be put out exactly in this sentence. Therefor, thank you for highlighting it!

Additionally, we are already indicating the vast options one can choose from with ROS2 here: https://github.com/asyncapi/bindings/blob/463fb3d3570cfcb51f5fe609ef348c343e0cbcf2/ros2/README.md?plain=1#L27